craft_application/project.py

```diff
@@ -265,8 +284,8 @@ def _dict_constructor(loader, node):
 
 
 class _SafeLoader(yaml.SafeLoader):  # pylint: disable=too-many-ancestors
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, stream: io.TextIOBase) -> None:
```

Comment:
hey does anyone know why we do this `_SafeLoader` loader?

Reply:
@sergiusens  I think you're probably best able to answer this

Reply:
Checked with Sergio and this prevents duplicate key entries. YAML 1.1 (as implemented by pyyaml) supports duplicate keys, 1.2 doesn't, but it also makes other backwards-incompatible changes.